whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -138,6 +137,14 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
             }
         }
 
+        if(objects.size() == maxResultSize) {
+            final Notice notice = new Notice();
+            notice.setTitle("Search result cap to " + maxResultSize);
```

Comment:
I suggest simplifying the text as 'limited search results to ... maximum' (s/capped/limited/g) and no need for details in the description.